e2e/tests/functional/plugins/faultManagement/faultManagement.e2e.spec.js

```diff
@@ -0,0 +1,165 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2022, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+const { test, expect } = require('../../../../pluginFixtures');
+const path = require('path');
+const { first } = require('lodash');
+
+test.describe('The Fault Management Plugin', () => {
+    test.beforeEach(async ({ page }) => {
+        await startAndNavigateToFaultManagement(page);
```

Comment:
I would use a new appAction for 'createDomainObjectWithDefaultValues'

Reply:
Yeah... good call, I'll set that up.

Reply:
Oh wait, we don't create this manually, it's created by the plugin itself, it's added as a root item. This wouldn't apply here.